src/ui/utils/comments.ts

```diff
@@ -36,14 +36,6 @@ export function formatRelativeTime(date: Date) {
   return "Now";
 }
 
-export function commentKeys(comments: (Comment | Reply)[]): string[] {
```

Comment:
if we are going to entirely get rid of these functions, I think you can delete `orderByApproximatelyCreatedAt`(was unused even before this) and `commentIdentifiers` as well.